### PR TITLE
API: get_fastccd_timestamps add fill=True

### PR DESCRIPTION
**Pull request by sameera2004**
tag=AreaDetectorHDF5TimestampHandler, changes due to get_events
function in the databroker

### CODE REVIEW

**Comment by sameera2004** [comment]
@stuwilkins, @cmazzoli please take a look at this function for retrieving timestamps. This is due to some changes in the get_events function in the data broker https://github.com/NSLS-II/databroker/blob/master/databroker/broker.py#L484 @tacaswell

File: csxtools/utils.py [hunk 1]
```diff
@@ -233,9 +234,9 @@ def get_fastccd_timestamps(header, tag='fccd_image'):
         list of arrays of the timestamps
 
     """
-    hover = {tag: AreaDetectorHDF5SWMRTimestampHandler}
+    hover = {tag: AreaDetectorHDF5TimestampHandler}
```
Comment (tacaswell) [inline csxtools/utils.py]: @stuwilkins Are you actually using swmr on the beamline?

**Comment by tacaswell** [comment]
How long have your travis builds been segfaulting @stuwilkins ?

**Comment by stuwilkins** [comment]
@tacaswell yes we have a SWMR compatable build at CSX. It was being tested, but not implemented for user DATA. No idea when TRAVIS started segfaulting.

**Comment by stuwilkins** [comment]
Superseded by #53

**Pull request closed without merging**